x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_columns/use_columns.ts

```diff
@@ -140,20 +144,40 @@ export const useColumns = ({
   storage,
   id,
   defaultColumns,
+  initialBrowserFields,
 }: UseColumnsArgs) => {
   const [isBrowserFieldDataLoading, browserFields] = useFetchBrowserFieldCapabilities({
     featureIds,
+    initialBrowserFields,
   });
+
   const [columns, setColumns] = useState<EuiDataGridColumn[]>(storageAlertsTable.current.columns);
   const [isColumnsPopulated, setColumnsPopulated] = useState<boolean>(false);
 
+  const defaultColumnsRef = useRef<typeof defaultColumns>(defaultColumns);
+
+  const didDefaultColumnChanged = useMemo(
```

Comment:
didDefaultColumnChange instead of didDefaultColumnChanged?

Reply:
Thanks. 👍